### ISSUE TITLE
IEEE Journal names abbreviated to a code

### ISSUE THREAD

**Issue by Ede123** [JabRef/jabref#2485]
#### Error description
It seems there's a problem in looking up the abbreviated name of IEEE journals.
For example abbreviating "IEEE Aerospace and Electronics Systems Magazine" results in "#IEEE_M_AES#"

#### JabRef version
```
JabRef 3.8.1
windows 7 6.1 x86 
Java 1.8.0_121
```

#### Steps to reproduce
1. Create an empty BibTeX database
2. Add an BibTeX entry (I used article)
3. Copy into the "Journal" field: "IEEE Aerospace and Electronics Systems Magazine"
4. Click "Toggle abbreviation"
5. Result: "[#IEEE_M_AES#](https://github.com/JabRef/jabref/blob/b1ec0e3c2cc24072cd0d0fbe6d233dbf8011c12d/src/main/resources/journals/IEEEJournalListCode.txt#L2)" as opposed to the expected "[IEEE Aerosp. Electron. Syst. Mag.](https://github.com/JabRef/jabref/blob/b1ec0e3c2cc24072cd0d0fbe6d233dbf8011c12d/src/main/resources/journals/IEEEJournalListText.txt#L3)"

#### Excerpt from the error console while doing this
(probably not relevant as the duplicates most likely come from an external file where I added the correct abbreviations for some journals as a workaround)
```
13:45:22.193 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.journals.JournalAbbreviationRepository - Duplicate journal abbreviation - old one will be overwritten by new one
OLD: Abbreviation{name=IEEE Journal of Quantum Electronics, iso=#IEEE_J_JQE#, medline=#IEEE_J_JQE#}
NEW: Abbreviation{name=IEEE Journal of Quantum Electronics, iso=IEEE J. Quantum Electron., medline=IEEE J Quantum Electron}
13:45:22.193 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.journals.JournalAbbreviationRepository - Duplicate journal abbreviation - old one will be overwritten by new one
OLD: Abbreviation{name=IEEE Journal of Selected Topics in Quantum Electronics, iso=#IEEE_J_JSTQE#, medline=#IEEE_J_JSTQE#}
NEW: Abbreviation{name=IEEE Journal of Selected Topics in Quantum Electronics, iso=IEEE J. Sel. Top. Quantum Electron., medline=IEEE J Sel Top Quantum Electron}
13:45:22.194 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.journals.JournalAbbreviationRepository - Duplicate journal abbreviation - old one will be overwritten by new one
OLD: Abbreviation{name=Journal of Lightwave Technology, iso=#IEEE_J_JLT#, medline=#IEEE_J_JLT#}
NEW: Abbreviation{name=Journal of Lightwave Technology, iso=J. Lightwave Technol., medline=J Lightwave Technol}
13:45:22.194 [AWT-EventQueue-0] INFO  net.sf.jabref.logic.journals.JournalAbbreviationRepository - Duplicate journal abbreviation - old one will be overwritten by new one
OLD: Abbreviation{name=Proceedings of the Institution of Electrical Engineers, iso=Proc. Inst. Electr. Eng., medline=Proc Inst Electr Eng}
NEW: Abbreviation{name=Proceedings of the Institution of Electrical Engineers, iso=Proc. Inst. Electr. Eng., medline=Proc Inst Electr Eng}
```

**Comment by Siedlerchr** [comment]
I found the solution for your problem. This is not a bug, but rather an unclear setting in the preferences.
Go to Preferences -> Advanced  and uncheck the Checkbox "use IEEE LaTeX Abbreviations". Than the second list is loaded and the journal names are displayed as in your example.

The LaTeX abbreviations are usually used in combination with http://tug.ctan.org/macros/latex/contrib/IEEEtran/bibtex/IEEEabrv.bib

**Comment by lenhard** [comment]
I'll interpret the missing updates here as the issue being solved. Therefore, I'll close it. Feel free to reopen if the solution explained by @Siedlerchr does not solve your problem.

**Comment by Ede123** [comment]
Hi, sorry for the delay but I wanted to actually check on the affected machine first (which is my work PC and I didn't get to it).

@Siedlerchr gave the correct hint. The checkbox was enabled and disabling fixed the issue.

I guess I must have switched it on while desperately trying to make it unabbreviate "IEEE J. Quantum Electron."  and "IEEE J. Sel. Top. Quantum Electron.". Obviously both couldn't work as the strings in `IEEEJournalListText.txt` read "IEEE J. Quantum. Electron." and "IEEE J. Sel. Topics in Quantum Electron." respectively (the first is a typo in the file and is fixed in #2579, the second I did probably mess up myself when adding an entry manually at an earlier point int time).
